### PR TITLE
Merge Project into Main: First major merge of my app

### LINKED ISSUE #1: [Story] Display basic projects
# Value statement

As a **user**
I need to **see projects**
so that **I can gather information on them**

# Description (Text, Scribble, Wireframe, Design)
image

-  Displays the most rudimentary possible, yet still usable project-cards to the user
-  Information includes a picture of the project, the project title and location, as well as a short description.

# Acceptance criteria

- [x] Displays multiple cards generated from a dataset 
- [x] Each card follows the design, no abbreviations
- [x] Pictures are same size
- [x] Cards have same width, slightly differing length due to text differences

# Tasks

- [x] Initialize React App
- [x] Create `Project` Component
- [x] Create `ProjectHeadline` Component (consisting of project-title and location)
- [x] Link image based from public dir
- [x] Create mock dataset (`db.json`) 
- [x] Render each element of the database (`.map()`)
- [ ] Tests

# Complexity
Small

### PR DESCRIPTION
**Pull request by constantingoeldel**
# Story #1 
This merge adds the functionality to display numerous projects as simple, non clickable cards. They are created from a db.json file filled with mock-data (Don't be irritated by the repeating pictures though, they solely serve as placeholders.) I'd appreciate a thorough review, especially when it comes to the cypress-tests! I didn't find a way to include jest-tests due to the lack of any data-manipulating functions so far, but maybe you can help me with that.

### CODE REVIEW

**Comment by maxbischof** [comment]
Did you think about adding propTypes?

**Comment by constantingoeldel** [comment]
> Did you think about adding propTypes?

I haven't, but I will include it soon. Thanks

File: src/components/ProjectHeadline.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default function ProjectHeadline({ title, country }) {
+  return (
+    <Headline>
```
Comment (karstenlenz) [inline src/components/ProjectHeadline.js]: Is it really necessary to wrap the h3 in a section? Do you really need a flex-container if it only contains one element?

**Review by karstenlenz — APPROVED** [review]
Look great! I only left 1 comment with a minor question.

**Comment by jamarob** [comment]
It is convenient to keep the cypress examples locally, but they are not needed in master. You can untrack them like this

1. `$ git rm -r --cached <path-to-cypress-examples>`
    This will remove the folder from the index, but keep your local copy. If you are unsure about this use the `--dry-run` flag to get a preview of what will be removed.
2. Add `<path-to-cypress-examples>` to your `.gitignore`
3. Commit and push again

✌️

File: src/components/ProjectHeadline.js (hunk 1 above)
Comment (constantingoeldel) [reply src/components/ProjectHeadline.js]: Thanks, changed it

**Comment by karstenlenz** [comment]
I did some QA testing on android (Chrome), and on MacOs (Chrome + Firefox). I couldn't find any bugs. However, in Chrome there is an error message in the console:
> Error while trying to use the following icon from the Manifest: https://unfinished.vercel.app/logo192.png (Download error or resource isn't a valid image)

You might want to have a look at that. Otherwise it looks fine.

**Pull request merged**